00-open-a-shapefile.Rmd

```diff
@@ -136,19 +134,24 @@ shapefiles include
 
 * a polygon shapefile representing our study area boundary, 
 * a line shapefile representing roads and streams, and 
-* a point shapefile representing the location of a [flux tower](http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements) located at the NEON 
-Harvard Forest field site.
+* a point shapefile representing the location of a [flux tower](http://www.neoninc.org/science-design/collection-methods/flux-tower-measurements) 
+located at the NEON Harvard Forest field site.
 
 The first shapefile that we will open, contains the boundary of our study area. 
 It is a polygon layer. We can tell it is a polygon by viewing it's `class` in
 the attributes of the object which is `SpatialPolygonsDataFrame`.
 
 ```{r Import-Shapefile}
+#set working directory to where the data files were saved.  
```

Comment:
Ok - now typically we have them create an R studio project file where the working directory is just set. MAYBE - we need to think about this - and it follows one of @jsta 's comments as well. 

I liked his idea of making things local so -- we write instructions for each lessons

1. download data from figshare
2. unzip data into a data folder on your computer (and make sure they unzip in the same way)
3. set working director to that location OR create project file. 

I did this for a workshop. let's chat about the best way to do this across ALL lessons.